underscore/underscore.d.ts

```diff
@@ -458,6 +458,10 @@ interface UnderscoreStatic {
 		object: _.Dictionary<T>,
 		iterator?: _.ObjectIterator<T, boolean>,
 		context?: any): boolean;
+		
```

Comment:
I would recommend to fix ln.449 to
```javascript
	any<T>(
		list: _.List<T>,
		predicate?: _.ListIterator<T, boolean> | T,
		context?: any): boolean;
```

And _.any is an alias for _.some.
_.some also needs to be fixed.